Schema/V1/metadatakatalog.xsd

```diff
@@ -960,6 +960,72 @@
     </xs:complexContent>
   </xs:complexType>
 
+  <xs:simpleType name="representerer">
+    <xs:annotation>
+      <xs:documentation>M374</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Ny datatype</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
+      <xs:minLength value="1"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:complexType name="moetesaksBeskrivelse">
+    <xs:annotation>
+      <xs:documentation>M375</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Nytt felt. Feltet tekstFormat er av typen kode med kodelisten: </xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element name="tekstInnhold" type="xs:string"/>
+      <xs:element name="tekstFormat" type="kode"/>
+    </xs:sequence>    
+  </xs:complexType>
+
+  <xs:simpleType name="behandlingsprotokoll">
+    <xs:annotation>
+      <xs:documentation>M376</xs:documentation>
+      <xs:documentation>Endret (fiks-arkiv): Nytt felt</xs:documentation>
+    </xs:annotation>
+    <xs:restriction base="xs:string">
```

Comment:
Dette feltet må være tilsvarende moetesaksBeskrivelse med mulighet for formatering.